### PR TITLE
Adds audit & Configuration page

### PR DESCRIPTION
**Pull request by sendilkumarn**

### CODE REVIEW

File: generators/client/templates/_package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "counterpart": "^0.17.2",
     "deep-equal": "^1.0.1",
     "flexboxgrid": "^6.3.1",
+    "fixed-data-table-2": "^0.7.12",
```
Comment (deepu105) [inline generators/client/templates/_package.json]: While the library is good, I dont think we need most of the features for our generated code. We use server side pagination and sorting. I was thinking of using http://www.material-ui.com/#/components/table

**Comment by sendilkumarn** [comment]
Rebased

File: generators/client/templates/_package.json (hunk 1 above)
Comment (sendilkumarn) [inline generators/client/templates/_package.json]: 👍

File: generators/client/templates/src/main/webapp/app/modules/administration/audits/audits.js [hunk 2]
```diff
@@ -2,12 +2,15 @@ import React, { Component } from 'react';
...
 
+import { Table, TableBody, TableHeader, TableHeaderColumn, TableRow, TableRowColumn } from 'material-ui/Table';
+
 import { getAudits } from '../../reducers/administration';
 
 export class AuditsPage extends Component {
 
   constructor(props) {
     super(props);
+    this.state = { showCheckboxes: false };
```
Comment (deepu105) [inline generators/client/templates/src/main/webapp/app/modules/administration/audits/audits.js]: I dont think this needs to be the component state.. we can set false on the declarations on the component declare it as  const in render method and use that. Ill merge that do that

File: generators/client/templates/src/main/webapp/app/modules/administration/audits/audits.js [hunk 3]
```diff
@@ -26,18 +29,33 @@ export class AuditsPage extends Component {
     return (
       <div className="well">
         <div>
-          <h2 translate="health.title">Audits</h2>
-          <p>
-            <button type="button" onClick={() => this.getAuditList()} className={isFetching ? 'btn btn-danger' : 'btn btn-primary'} disabled={isFetching}>
```
Comment (deepu105) [inline generators/client/templates/src/main/webapp/app/modules/administration/audits/audits.js]: why did you remove this? I think its needed once we add the input fields

Comment (deepu105) [inline generators/client/templates/src/main/webapp/app/modules/administration/audits/audits.js]: Oh ok. I didn't check that

**Comment by deepu105** [comment]
Merge it when you are done

**Comment by sendilkumarn** [comment]
I am merging this and progressively merge others
